src/AppInstallerCLICore/Commands/COMInstallCommand.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "Command.h"
+
+namespace AppInstaller::CLI
+{
+    struct COMInstallCommand final : public Command
```

Comment:
> COMInstallCommand  [](http://example.com/codeflow?start=10&length=19)

I think we should better extend the current Install command(and other commands COM supports in the future, but not now) to have a flag indicating "it's from COM" rather than creating a duplicate command here.

By the way, I need that flag too with my current work to support source agreements

Reply:
or maybe even simpler, something like IsCom() exposed in context if it works

Reply:
I disagree; this type of large scale context sensitivity is what we want to avoid with workflows.  I suggested potentially making a different base object to hold the concept of a composed workflow though.

I'm not following the part about source agreements though.

Reply:
Chatted with John offline, I was not aware of the required differences between COM workflow and the regular flow. I got the wrong first impression by the name ComInstallCommand, instead it's sort of a different set of workflow tasks.

I think we should clean up any nonapplicable code from this file and maybe name it differently?

Regarding source agreements, we'll use a specific flag say SKIP_AGREEMENTS in context flags.

Reply:
@JohnMcPMS, @yao-msft,  does the more precise InstallFromManifestCommand suit better for the name ?